react/features/chat/components/web/ChatInput.tsx

```diff
@@ -147,7 +157,9 @@ class ChatInput extends Component<IProps, IState> {
                         accessibilityLabel = { this.props.t('chat.sendButton') }
                         disabled = { !this.state.message.trim() }
                         icon = { IconSend }
-                        onClick = { this._onSubmitMessage }
+                        onClick={() => {
+                            this._onSubmitMessage() 
+                            this._toggleSmileysPanel();}}
```

Comment:
Is this correct? Seems strange to toggle on/off the smiley panel each time a message is sent.